Makefile

```diff
@@ -64,6 +68,13 @@ logs: ## display grafana logs (follow mode)
 	@$(COMPOSE) logs -f grafana
 .PHONY: logs
 
+plugins: ## download plugins
+	mkdir -p ./var/lib/grafana/plugins
```

Comment:
I would add a `var/lib/grafana/plugins` rule in the FILES section, add this rule to the tree rule dependencies and make the `plugins` rule depend on the `tree` rule.

Reply:
Good Idea) Thanks) Updated this part) the `var/lib/grafana/plugins` rule should just create the plugins directory ?

Reply:
Yes.